src/pages/index.js

```diff
@@ -45,45 +50,133 @@ const handleCardClick = (cardData) => {
   popupWithImage.open(cardData.link, cardData.name);
 };
 
-const createCard = (cardData) => {
-  const card = new Card(cardData, "#card-template", handleCardClick);
+const handleDeleteClick = (cardData, cardEl) => {
+  console.log("handleDeleteClick", deleteConfirmationPopup);
+  deleteConfirmationPopup.open();
+  console.log(cardEl);
+  deleteConfirmationPopup.setId(cardData._id);
+  deleteConfirmationPopup.setCard(cardEl);
+  console.log(deleteConfirmationPopup.getId());
+};
+
+const handleLikeCard = (card) => {
+  console.log(card);
+  api.toggleLikeOnCard(card._cardData._id, card._isLiked).then((response) => {
```

Comment:
1) private variables should not be used outside of a class, you can create a getter for id `getId` in Card class
2) use card.isLiked()